apis/hive/v1/clusterdeploymentcustomization_types.go

```diff
@@ -54,9 +57,50 @@ type PatchEntity struct {
 	// From is the json path to copy or move the value from
 	// +optional
 	From string `json:"from,omitempty"`
-	// Value is the value to be used in the operation
-	// +required
-	Value string `json:"value"`
+	// Value is the *string* value to be used in the operation. For more complex values, use
+	// ValueJSON.
+	// +optional
+	Value string `json:"value,omitempty"`
+	// ValueJSON is a string representing a JSON object to be used in the operation. As such,
+	// internal quotes must be escaped. If nonempty, Value is ignored.
+	// +optional
+	ValueJSON string `json:"valueJSON,omitempty"`
```

Comment:
Should we make this `[]byte` rather than `string`?

Reply:
string is fine I'd say